.github/workflows/pages-pr.yml

```
@@ -0,0 +1,40 @@
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        working-directory: docs
```

actionlint (1.7.7)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)